### PR TITLE
Web page clean ups and bug fixes 

### LINKED ISSUE #731: Drop footer menu from news and main
Following up on https://github.com/OpenSourcePolicyCenter/blog-posts/pull/11, I think the right next step is to:

Remove from ospc.org:
image

Remove from news.ospc.org:
image

cc @hdoupe @jessmc @mikejpratt

### PR DESCRIPTION
**Pull request by GoFroggyRun**
This PR closes #731 #732 #737.

### CODE REVIEW

**Comment by GoFroggyRun** [comment]
@hdoupe can we have this PR included in the new update?

**Comment by hdoupe** [comment]
To drop the footer from the news.ospc.org page, you will have to edit the [blog-posts repo](https://github.com/OpenSourcePolicyCenter/blog-posts).  This PR just drops the footer from the pages in this repo.

It appears that #737 isn't fully resolved either since the link https://ospc.org/hello still hangs.  As @MaxGhenis pointed out in #737 it should be https://www.ospc.org/hello/.

Once #737 is resolved, we can merge this.  I'll open an issue for this in the blog-posts repo.

File: templates/pages/docs.html [hunk 1]
```diff
@@ -33,12 +33,12 @@ <h2>Overview</h2>
       <li><strong>Step 6.</strong> Share your results! The link to every results page is static and will never change. Send them around.</li>
       </ul>
 
-      <p> Throughout this process, if you have a question about how to use TaxBrain or interpret the results, if you want to make a suggestion for making the interface or underlying models better, or if you discover a bug, please <a href = '//list.ospc.org/mailman/listinfo/users_list.ospc.org' target="_blank">join our mailing list</a> and send a message or <a href = '//ospc.org/hello' target="_blank">leave a note for the OSPC team</a>.</p>
+      <p> Throughout this process, if you have a question about how to use TaxBrain or interpret the results, if you want to make a suggestion for making the interface or underlying models better, or if you discover a bug, please <a href = '//list.ospc.org/mailman/listinfo/users_list.ospc.org' target="_blank">join our mailing list</a> and send a message or <a href = '//www.ospc.org/hello' target="_blank">leave a note for the OSPC team</a>.</p>
```
Comment (GoFroggyRun) [inline templates/pages/docs.html]: @hdoupe won't this line fix #737?

Comment (hdoupe) [reply templates/pages/docs.html]: I just tested it and it didn't seem to work correctly.  I'll take another look though.

Comment (GoFroggyRun) [reply templates/pages/docs.html]: Strange. Both links are working on my local.

Comment (hdoupe) [reply templates/pages/docs.html]: Just checked again, and it worked.  Sorry about that.

**Comment by hdoupe** [comment]
Looks good to me. Thanks @GoFroggyRun

**Pull request merged**